source/nodejs/adaptivecards/src/adaptivecards.ts

```diff
@@ -1,5 +1,5 @@
 export * from "./card-elements";
 export * from "./enums";
 export * from "./host-config";
-export { IAdaptiveCard, ICardElement } from "./schema";
+export { IAdaptiveCard, ICardElement, Spacing, HorizontalAlignment } from "./schema";
```

Comment:
There is no need to export these two enums given they already exist in enums.ts

Reply:
Fixed